supplements/extra-life/adventure-with-muk.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Adventure with Muk</name>
+		<update version="0.0.1">
+			<file name="adventure-with-muk.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/extra-life/adventure-with-muk.xml" />
+		</update>
+	</info>
+
+	<element name="Adventure with Muk" type="Source" source="Core" id="ID_SOURCE_ADVENTURE_WITH_MUK">
+		<description>
+			<p>Meet Muk, the very bravest goblin, and his best friend Birdsquirrel. Adventure your way through the Dankwood and search for a great treasure that is rumored to be hidden far beyond.</p>
+			<p>For players of all levels.</p>
+			<p>This activity book includes enjoyable adventure hooks written by D&amp;D’s Adam Lee, puzzles, unique Dankwood goblin character sheets developed by D&amp;D’s Chris Lindsay, modified monster stat blocks and more! Each page is made to be colored and enjoyed, so let the adventure begin!</p>
+			<p>Best of all, ALL monies that Wizards of the Coast receives from sales of this PDF are donated to Extra Life. Your purchase of this adventure goes to a truly great cause!</p>
+		</description>
+		<setters>
+			<set name="abbreviation">AWM</set>
+			<set name="url">https://www.dmsguild.com/product/294712/Adventure-with-Muk-5e</set>
+			<set name="image">https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/extra-life/EL-AWM.jpg</set>
```

Comment:
since there's no such image, it is up to this pull to add it (you don't need to add it to the index though, since it's just an online reference).

Reply:
Added it. Apparently I forgot to upload the file before.